cost-analyzer/values.yaml

```diff
@@ -2450,64 +2450,50 @@ kubecostAggregator:
   # after consulting with Kubecost's support team
   numDBCopyPartitions: 25
 
-  resources: {}
-    # requests:
-    #   cpu: 1000m
-    #   memory: 1Gi
   env:
     "LOG_LEVEL": "info"
     "DB_READ_THREADS": "1"
     "DB_WRITE_THREADS": "1"
     "DB_CONCURRENT_INGESTION_COUNT": "3"
+
   persistentConfigsStorage:
-    # default storage class
-    storageClass: ""
+    storageClass: ""  # default storage class
     storageRequest: 1Gi
   aggregatorDbStorage:
-    # default storage class
-    storageClass: ""
+    storageClass: ""  # default storage class
     storageRequest: 128Gi
 
+  resources: {}
+    # requests:
+    #   cpu: 1000m
+    #   memory: 1Gi
+
   readinessProbe:
     enabled: true
     initialDelaySeconds: 10
     periodSeconds: 10
     failureThreshold: 200
 
+  ## Set additional environment variables for the aggregator pod
+  # extraEnv:
+  # - name: SOME_VARIABLE
+  #   value: "some_value"
+
   ## Add a priority class to the aggregator pod
   # priority:
   #   enabled: false
   #   name: ""
 
-  # extraEnv:
-  # - name: SOME_VARIABLE
-  #   value: "some_value"
-  # securityContext:
-  #   runAsGroup: 1001
-  #   runAsUser: 1001
-  #   fsGroup: 1001
-  #   fsGroupChangePolicy: OnRootMismatch
-  #   seccompProfile:
-  #     type: RuntimeDefault
-  #   runAsNonRoot: true
-  # containerSecurityContext:
-  #   allowPrivilegeEscalation: false
-  #   readOnlyRootFilesystem: true
-  #   runAsNonRoot: true
-  #   seccompProfile:
-  #     type: RuntimeDefault
-  #   capabilities:
-  #     drop:
-  #       - ALL
```

Comment:
Deleting this comment because we already have `securityContext` and `containerSecurityContext` commented below.